lib/matplotlib/tests/test_ticker.py

```diff
@@ -590,6 +624,25 @@ def test_values(self, vmin, vmax, expected):
         ticks = sym.tick_values(vmin=vmin, vmax=vmax)
         assert_array_equal(ticks, expected)
 
+    def test_subs(self):
+        fig, ax = plt.subplots()
+        sym = mticker.SymmetricalLogLocator(base=10, linthresh=1, subs=[2.0, 4.0])
+        ax.xaxis.set_major_locator(sym)
+        ax.set_xlim(-10, 10)
+        assert (sym() == [-20., -40.,  -2.,  -4.,   0.,   2.,   4.,  20.,  40.]).all()
```

Comment:
Why do the negative ones seem to be in reverse order within each decade?

Reply:
The current code works this way?

I was thinking of adding a `sorted` here, but was waiting to get some comments before actually changing something. (All changes so far do not change any output.)

Reply:
Better to not change the code?